src/global.ts

```diff
@@ -83,11 +94,21 @@ export default class Global extends Thread {
 
     // To allow library users to specify custom types
     // Higher is more important and will be evaluated first.
+    /**
+     * Registers a type extension for Lua objects.
```

Comment:
There is some comment above, can we merge them?

Reply:
I did want to merge the "Higher is more important and will be evaluated first.", as for the first line that seems to be more of a comment to library maintainers

Reply:
Something along the lines of 
```js
/**
 * Registers a type extension for Lua objects.
 * Higher priority is evaluated first
 */